recipes-alexa/avs-device-sdk/avs-device-sdk.inc

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
Isn't that 'our' recipe? If the recipe is coming from outside then it is available on official meta layer. In such case we shouldn't import the recipe to our layers, and include official meta layer with that recipe in our builds